src/platforms/dotnet/guides/aspnetcore/index.mdx

```diff
@@ -140,43 +93,71 @@ let CreateHostBuilder args =
         )
 ```
 
-ASP.NET Core 6.0 With minimal API:
+### ASP.NET Core 2.x
 
 ```csharp
-var builder = WebApplication.CreateBuilder(args);
+public static IWebHost BuildWebHost(string[] args) =>
+    WebHost.CreateDefaultBuilder(args)
+        // Add the following line:
+        .UseSentry()
+```
 
-builder.WebHost.UseSentry();
+```fsharp
+let CreateHostBuilder args =
+    WebHost.CreateDefaultBuilder(args)
+        // Add the following line:
+        .UseSentry()
+```
 
-var app = builder.Build();
+## Configure
 
-app.UseSentryTracing();
-```
+The framework takes configuration settings from `appsettings.json`, environment variables or from code. These settings get applied to `Sentry.AspNetCore.SentryAspNetCoreOptions`, which extends `Sentry.Extensions.Logging.SentryLoggingOptions`, which in turn extends `Sentry.SentryOptions`. This means options for `Sentry.AspNetCore` as well as all the inner packages are available to be configured through the configuration system.
 
-```fsharp
-open Microsoft.AspNetCore.Builder
-open Microsoft.Extensions.Hosting
-open Microsoft.AspNetCore.Hosting
+### appsettings.json
 
-[<EntryPoint>]
-let main args =
-    let builder = WebApplication.CreateBuilder(args)
-    builder.WebHost.UseSentry() // you'll need to include Microsoft.AspNetCore.Hosting.
+An example of some of the options that can be configured via `appsettings.json`:
 
-    let app = builder.Build()
-    app.UseSentryTracing()
+<SignInNote />
 
-    app.Run()
+```json {filename:appsettings.json}
+  "Sentry": {
+    "Dsn": "___PUBLIC_DSN___",
+    "SendDefaultPii": true,
+    "MaxRequestBodySize": "Always",
+    "MinimumBreadcrumbLevel": "Debug",
+    "MinimumEventLevel": "Warning",
+    "AttachStackTrace": true,
+    "Debug": true,
+    "DiagnosticsLevel": "Error",
+    "TracesSampleRate": 1.0
+  },
+```
 
-    0 // Exit code
+Note setting `TracesSampleRate` to `1.0` in this example configures Sentry to capture 100% of transactions for performance monitoring. We recommend adjusting this value in production. See <PlatformLink to="/performance/instrumentation/automatic-instrumentation">Automatic Instrumentation</PlatformLink> for further information.
+
+### Environment variables
+
+It's also possible to bind properties to the SDK via environment variables, like:
+
+```shell {tabTitle:Windows}
+set Sentry__Debug=true
+```
 
+```shell {tabTitle:Linux or macOS}
+export Sentry__Debug=true
```

Comment:
Was this supposed to have 2 underscores?

```suggestion
export Sentry_Debug=true
```

Reply:
> Was this supposed to have 2 underscores?

Hey @shanamatthews,

Yes it is... the .NET configuration system uses double underscores instead of `:` when setting configuration properties from environment variables ([see docs](https://learn.microsoft.com/en-us/aspnet/core/fundamentals/configuration/?view=aspnetcore-7.0#evcp)):

> The `:` separator doesn't work with environment variable hierarchical keys on all platforms. `__`, the double underscore, is:
>
> * Supported by all platforms. For example, the `:` separator is not supported by [Bash](https://linuxhint.com/bash-environment-variables/), but `__` is.
> * Automatically replaced by a `:`